src/pages/grants.tsx

```diff
@@ -106,46 +112,76 @@ export default function GrantsPage() {
     }
   }, [grants, selectedCategory])
 
-  if (isUnderMaintenance()) {
-    return (
-      <>
-        <Head
-          title={t('page.grants.title') || ''}
-          description={t('page.grants.description') || ''}
-          image="https://decentraland.org/images/decentraland.png"
-        />
-        <Navigation activeTab={NavigationTab.Grants} />
-        <MaintenancePage />
-      </>
-    )
-  }
-
-  const isLoading = isEmpty(grants) && isLoadingGrants
-
-  const getCurrentBannerItems = () => {
-    if (isLoading) {
+  const getPastBannerItems = () => {
+    if (isEmpty(grants)) {
       return []
     }
 
+    const totalProjects = grants.past.length
+    const sizes = grants.past.map((item) => item.configuration.size)
+    const totalFunding = sizes.reduce((prev, next) => prev + next, 0)
+    const approvedPercentage = Math.round(((grants.current.length + grants.past.length) * 100) / grants.total)
+
     return [
-      { title: `${grants.current.length} projects open`, description: 'Initiatives currently being funded' },
-      { title: '$2.5 million USD released', description: 'Funding so far, for current batch' },
-      { title: '$1.3 million USD to go', description: 'To be released for current batch' },
+      { title: `${totalProjects} Projects`, description: 'Successfully completed Grants' },
+      {
+        title: `$${formatter.format(totalFunding)} USD`,
+        description: 'Total funding for past initiatives',
+      },
+      { title: `${approvedPercentage}% Approval`, description: 'Rate of approved Grant proposals' },
     ]
   }
 
-  const getPastBannerItems = () => {
-    if (isLoading) {
+  const isLoading = isEmpty(grants) && isLoadingGrants
+
+  const getCurrentBannerItems = () => {
+    if (isEmpty(grants)) {
       return []
     }
 
+    const releasedValues = grants.current.map((item) => {
+      if (item.configuration.tier === 'Tier 1' || item.configuration.tier === 'Tier 2') {
+        return item.configuration.size
+      }
+
+      const releasedPercentage = ((item.contract?.released || 0) * 100) / (item.contract?.vesting_total_amount || 0)
+
+      return ((item.contract?.vesting_total_amount || 0) * releasedPercentage) / 100
```

Comment:
I think it should be the `configuration.size` instead of the `vesting_total_amount`, right? So we get it in USD